extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -67,6 +74,17 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		return this._initializeDeferred.promise;
 	}
 
+	private async initializePinnedNotebooks(): Promise<void> {
+		await Promise.all(getPinnedNotebooks().map(async (notebookPath) => {
+			try {
+				await this.createAndAddBookModel(notebookPath, true);
+			} catch {
+				// no-op, not all workspace folders are going to be valid books
+			}
+		}));
+		this._initializeDeferred.resolve();
```

Comment:
You should only be calling this once - you're already calling it in intialize.

Would suggest moving the calling of this function into initialize